Makefile

```diff
@@ -14,7 +14,7 @@ server: hugo
 	./hugo server -w -s src
 
 static: hugo
-	./hugo -D -s src -b $(OUTPUT_DIR) -d ../output/$(OUTPUT_DIR)
+	./hugo -D -s src -b $(OUTPUT_DIR) -d ../output$(OUTPUT_DIR)
```

Comment:
This doesn’t make any difference, because the Makefile read at this point when processing `static-all` is the Makefile in the “other” branch...

Reply:
Ack, but it's a cherry-pick backport. If we want to change this we must do it first on the master branch.

and btw, I know it doesn't make a difference, but I thought it made the script more readable.

Reply:
Agreed, and note that I approved the patch without requesting changes. I just noticed the behaviour when reviewing the patch, which surprised me, so I thought I’d note it in the PR.

Reply:
Ah... sorry, I didn't make it more readable, I made it less readable.. :P 

I will propose another patch to master...

Reply:
https://github.com/submariner-io/submariner-website/pull/423